server/urls.py

```diff
@@ -8,6 +8,10 @@
 admin.autodiscover()
 
 urlpatterns = [
+    # user urls
+    url(r'^users/$', server.views.user, name='user'),
+    url(r'^users/(?P<name>[a-zA-Z0-9]+)', server.views.user, name='user'),
```

Comment:
On second thought, how about we skip the /users/ part of this? Github. Less clutter this way.